charts/docker-minio/README.md

```diff
@@ -0,0 +1,50 @@
+# docker-minio
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: latest](https://img.shields.io/badge/AppVersion-latest-informational?style=flat-square)
+
+A Helm chart for deploying SparkFabrik Docker-MinIO application on Kubernetes.
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| image.registry | string | `"ghcr.io"` |  |
+| image.repository | string | `"sparkfabrik/docker-minio"` |  |
+| image.pullPolicy | string | `"IfNotPresent"` |  |
+| image.tag | string | `""` |  |
+| imagePullSecrets | list | `[]` |  |
+| nameOverride | string | `""` |  |
+| fullnameOverride | string | `""` |  |
+| env | object | `{}` |  |
+| envFrom | string | `nil` |  |
+| serviceAccount.create | bool | `true` |  |
+| serviceAccount.automount | bool | `true` |  |
+| serviceAccount.annotations | object | `{}` |  |
+| serviceAccount.name | string | `""` |  |
+| podAnnotations | object | `{}` |  |
+| podLabels | object | `{}` |  |
+| podSecurityContext | object | `{}` |  |
+| securityContext | object | `{}` |  |
+| service.type | string | `"ClusterIP"` |  |
+| service.port | int | `9000` |  |
+| ingress.enabled | bool | `false` |  |
+| ingress.className | string | `""` |  |
+| ingress.annotations | object | `{}` |  |
+| ingress.hosts[0].host | string | `"chart-example.local"` |  |
+| ingress.hosts[0].paths[0].path | string | `"/"` |  |
+| ingress.hosts[0].paths[0].pathType | string | `"ImplementationSpecific"` |  |
+| ingress.tls | list | `[]` |  |
+| resources.requests.memory | string | `"128Mi"` |  |
+| resources.requests.cpu | string | `"50m"` |  |
+| livenessProbe.httpGet.path | string | `"/minio/health/live"` |  |
+| livenessProbe.httpGet.port | string | `"http"` |  |
+| readinessProbe.httpGet.path | string | `"/minio/health/live"` |  |
+| readinessProbe.httpGet.port | string | `"http"` |  |
+| volumes | list | `[]` |  |
+| volumeMounts | list | `[]` |  |
+| nodeSelector | object | `{}` |  |
+| tolerations | list | `[]` |  |
+| affinity | object | `{}` |  |
+
```

Comment:
How do we manage the console port?

Reply:
Added [here](https://github.com/sparkfabrik/helm-charts/pull/5/files#diff-e642187f173db9fa435db4b11b533ff56e256bc31cac59d553a414a6bd37f3bfR20).